rasa_core/utils.py

```diff
@@ -17,6 +17,10 @@
 from builtins import input, range, str
 from numpy import all, array
 from typing import Text, Any, List, Optional, Tuple, Dict, Set
+try:
```

Comment:
is this a py2 / py3 thing? can we use `six` instead?

Reply:
it is, use six how?

Reply:
saw your other comment, ignore me!